app/src/main/java/com/framgia/forder/data/model/DataSuggest.java

```diff
@@ -0,0 +1,85 @@
+package com.framgia.forder.data.model;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import com.arlib.floatingsearchview.suggestions.model.SearchSuggestion;
+import io.realm.RealmObject;
+
+/**
+ * Created by Age on 4/27/2017.
+ */
+
+@SuppressWarnings("DefaultFileTemplate")
+public class DataSuggest extends RealmObject implements SearchSuggestion, Parcelable {
+    private boolean mIsHistory = false;
+    private String mName;
+    private int mDomainId;
+
+    public DataSuggest() {
+    }
+
+    public DataSuggest(int domainId, String name) {
+        mDomainId = domainId;
+        mName = name;
+    }
+
+    private DataSuggest(Parcel in) {
+        mDomainId = in.readInt();
+        mName = in.readString();
+    }
+
+    public static final Parcelable.Creator<DataSuggest> CREATOR =
+            new Parcelable.Creator<DataSuggest>() {
+                public DataSuggest createFromParcel(Parcel in) {
+                    return new DataSuggest(in);
+                }
+
+                public DataSuggest[] newArray(int size) {
+                    return new DataSuggest[size];
+                }
+            };
+
+    public DataSuggest(String name) {
+        mName = name;
+    }
+
+    public boolean getHistory() {
```

Comment:
WARNING: [BooleanGetMethodName] A getX() method which returns a boolean should be named isX()
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#BooleanGetMethodName